templates/application/lit-html/base/rspack.config.js

```diff
@@ -1,6 +1,38 @@
 const rspack = require('@rspack/core')
 const refreshPlugin = require('@rspack/plugin-react-refresh')
 const isDev = process.env.NODE_ENV === 'development'
+const path = require('path');
+
+const printCompilationMessage = (status, port) => {
+  let messageColor, messageType, browserMessage
+
+  switch (status) {
+    case 'success':
+      messageColor = '\x1b[32m'
+      messageType = 'Compiled successfully!'
+      browserMessage = 'You can now view'
+      break
+    case 'failure':
+      messageColor = '\x1b[31m'
+      messageType = 'Compilation Failed!'
+      browserMessage = 'You can\'t now view'
+      break
+    case 'compiling':
+      messageColor = '\x1b[94m'
+      messageType = 'Compiling...'
+      browserMessage = 'Compiling the'
+      break
+  }
```

Comment:
Can it be possible to extract this function into another higher level to be able to reuse it in the other 15 places? 
![Screenshot 2024-07-01 at 08 52 10](https://github.com/jherr/create-mf-app/assets/5256287/e9a452a7-73cd-4387-a609-410aba51dcc8)

Reply:
Thanks for the idea!
I think this covers it 🥰🥰 https://github.com/jherr/create-mf-app/pull/69#issuecomment-2199946630